RDF-Pipeline/t/tests/0049_DependsOn_clock_site/setup-files/node/pipeline.ttl

```diff
@@ -5,5 +5,5 @@
 @prefix : <http://localhost/node/> .
 
 :hello a p:FileNode ;
-  p:dependsOn <http://www.time.gov/timezone.cgi?Eastern/d/-5> .
+  p:dependsOn <http://www.time.gov> .
```

Comment:
But the Date header cannot be used for this purpose, because it only indicates when the server responded -- not when the response content was last modified:
https://tools.ietf.org/html/rfc7231#section-7.1.1.2